database/queries/messages_code/create.sql

```diff
@@ -3,8 +3,8 @@ CREATE TABLE IF NOT EXISTS messages_code
   id SERIAL PRIMARY KEY,
   user1_Id INT,
   user2_Id INT,
-  room_Id VARCHAR(40) UNIQUE NOT NULL,
+  room_Id VARCHAR(40) NOT NULL,
```

Comment:
We definitely do want unique room ids, just not on this table. Room Id's will be unique on the rooms table, but the messages_code join table will have potentially have an unlimited number of messages sent on the same room, and we would only be able to save 1 message per room if we force the id to be unique in this table.